lonestar/analytics/cpu/betweennesscentrality/README.md

```diff
@@ -59,32 +62,37 @@ Pass in a regular .gr graph.
 For more details on the algorithm, see paper here:
 https://dl.acm.org/citation.cfm?id=2442521
 
+INPUT
```

Comment:
Since both algorithms are taking the same input and build instructions are the same; These repetitions can be removed.